util/testutil/testhelpers/minio.go

```diff
@@ -114,3 +116,13 @@ func waitMinio(address string, d time.Duration) error {
 	}
 	return nil
 }
+
+func randomString(n int) string {
```

Comment:
Not related to these changes but this is used by minio and we could just remove this func and use `identity.NewID()` instead.